official/resnet/resnet.py

```diff
@@ -278,8 +370,9 @@ class Model(object):
 
   def __init__(self, resnet_size, num_classes, num_filters, kernel_size,
                conv_stride, first_pool_size, first_pool_stride,
-               second_pool_size, second_pool_stride, block_fn, block_sizes,
-               block_strides, final_size, data_format=None):
+               second_pool_size, second_pool_stride, block_sizes, block_strides,
+               final_size, version=DEFAULT_VERSION, bottleneck=False,
```

Comment:
nit: can we keep the original ordering of the bottleneck arg? I had originally put them in order of use, so bottleneck would belong in the middle there.

Reply:
Python doesn't allow args without defaults to follow those with, and I do like having defaults on version and bottleneck.

Reply:
I would argue with so many other params, and a well-defined set of callers of this, we should just remove the default. Especially since the default run with current command line args actually does use bottleneck blocks.

Reply:
For bottleneck. Version we should leave.

Reply:
Ok. Changed.